app/models/foreman_openscap/arf_report.rb

```diff
@@ -5,6 +5,7 @@ class ArfReport < ::Report
     include Taxonomix
     include OpenscapProxyExtensions
 
+    # attr_accessible :host_id, :reported_at, :status, :metrics
```

Comment:
This was fixed with https://github.com/theforeman/foreman/pull/2961
fetch & rebase from foreman/develop :)